src/main/resources/hudson/security/table.js

```diff
@@ -215,11 +231,16 @@ Behaviour.specify(".global-matrix-authorization-strategy-table td input", 'Globa
       if (reference.checked) {
         e.disabled = true;
         let tooltip = YAHOO.lang.escapeHTML(findAncestor(e, "TD").getAttribute('data-tooltip-disabled'));
-        e.setAttribute('tooltip', tooltip); // before 2.335 -- TODO remove once baseline is new enough
-        e.nextSibling.setAttribute('tooltip', tooltip); // 2.335+
+        e.setAttribute(tooltipAttributeName, tooltip); // before 2.335 -- TODO remove once baseline is new enough
+        e.nextSibling.setAttribute(tooltipAttributeName, tooltip); // 2.335+
       }
     }
   }
+
+  if (window.registerTooltips) {
+    window.registerTooltips(e.nextSibling.parentElement);
+  }
```

Comment:
What was the purpose of this? It doesn't work since https://github.com/jenkinsci/jenkins/pull/7464 renamed the method, and the other occurrence of `registerTooltips` was fixed up in https://github.com/jenkinsci/matrix-auth-plugin/pull/130, but this code still exists.

Reply:
I think it was fixed in core itself with behaviour and then applying subtree here.

the purpose was so that when checkboxes were dynamically added they would have tippy run on this but it was fixed in another way...